### PR TITLE
Re arrange videos

### PR DESCRIPTION
**Pull request by JeanVegaD**
### GH Issue

re-arrange videos #58 

**`Productor de Bloques`**

1.  Move the content to `Mecanismos de consenso`
1. Delete `Productor de Bloques` page

**`DevOps`**

1. Delete broken links

**`Cadena de bloques EOS`**

1. Delete broken links
1. Move "Brief introduction to c++" video to `c++ moderno` 
1. Move "Basic EOSIO smart contract schema...." video to `contratos inteligentes`
1. Delete duplicate content from  `contratos inteligentes`
1. Add link to `contratos inteligentes` page

 **`Software EOS`** 

1. Move the video to `cuentas y permisos`
1. Move "Delegated Proof of Stake (DPoS)"  information to `mecanismos de concenso`
1. Delete `Software EOS` page

### Steps to test
1. Run project locally
1. Open the website
1. Look the content in "Aprender EOSIO" tab

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/aprender-eosio/desarrollo-de-dapps.md [hunk 1]
```diff
@@ -1,22 +1,17 @@
 ---
 id: desarrollo-de-dapps
-title: Desarrollo de Daps
+title: Desarrollo de Dapps
 sidebar_label: Desarollo de Dapps
 ---
 
-## Construyendo Aplicaciones Descentralizadas
+## Porqué EOS es construido para Aplicaciones Descentralizadas
```
Comment (JustinCast) [inline docs/aprender-eosio/desarrollo-de-dapps.md:7]: @xavier506 @JeanVegaD maybe we can say "usado" or "fue diseñado"

**Comment by JustinCast** [comment]
@JeanVegaD please fix the conflicts in `website/sidebars.json`

File: docs/aprender-eosio/desarrollo-de-dapps.md (hunk 1 above)
Comment (xavier506) [reply docs/aprender-eosio/desarrollo-de-dapps.md:7]: ## EOSIO para el desarrollo de aplicaciones descentralizadas (DApps) ?

File: docs/aprender-eosio/desarrollo-de-dapps.md [hunk 2]
```diff
@@ -31,38 +26,8 @@ Puedes ver más en este canal: https://www.youtube.com/user/apexmaxable/videos
 
 ## Empezando
 
-https://developers.eos.io/eosio-cpp/docs/introduction-to-smart-contracts
```
Comment (xavier506) [inline docs/aprender-eosio/desarrollo-de-dapps.md:34]: @JeanVegaD  porque estamos eliminando este contenido?

Comment (JeanVegaD) [reply docs/aprender-eosio/desarrollo-de-dapps.md:34]: @xavier506  Es contenido duplicado sobre `contratos-inteligentes` que ya se encuentra disponible, por lo que se hace referencia a esta pagina  [https://guias.eoscostarica.io/docs/aprender-eosio/contratos-inteligentes](https://guias.eoscostarica.io/docs/aprender-eosio/contratos-inteligentes).

**Pull request merged**